axelrod/tests/strategies/test_backstabber.py

```diff
@@ -172,101 +167,3 @@ def test_alt_strategy_stops_after_round_180(self):
             expected_actions=expected_actions,
             match_attributes={"length": 200},
         )
-
-
-class TestModuleMethods(unittest.TestCase):
```

Comment:
Why can we remove all of these?

Reply:
Good question. Some are just testing Player class properties that already have their own tests. But it looks like I was a bit overzealous.

Reply:
Actually I think all of these tests are redundant. If we want to keep any I'd prefer we re-write them to use an actual match rather than simply overwriting the history, which is a practice that we shouldn't encourage.

Reply:
Yeah I agree about not encouraging using history overwriting, could we either rewrite as part of this PR or leave them in and open an issue to remove them?

Reply:
I'll check but I think these tests are just targeting the subfuctions of the strategy and are all covered but the "standard" tests above in the file.